internal/testing/node.go

```diff
@@ -37,24 +38,37 @@ func DefaultConfig(net config.NetworkType, node config.NodeType, netId string) *
 func NodeInitOptsForNetwork(subnet *networks.Subnet) node.InitOptions {
 	listenIP := make([]string, len(subnet.Nodes))
 	remoteIP := make([]string, len(subnet.Nodes))
+	var listener string
 	cfg := make([]*config.Config, len(subnet.Nodes))
 
 	for i, net := range subnet.Nodes {
+		listener = "unix:///tmp/accumulate.sock"
 		listenIP[i] = "localhost"
 		remoteIP[i] = net.IP
 		cfg[i] = DefaultConfig(subnet.Type, net.Type, subnet.Name) // Configure
 	}
+	var la *net.UnixAddr
+	la = &net.UnixAddr{
+		Name: listener,
+		Net:  "unix",
+	}
+	ks, err := net.ListenUnix("unix", la)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Why are we starting to listen here? All this function does is generate configuration.